pdata/internal/cmd/pdatagen/internal/pprofile_package.go

```diff
@@ -171,17 +171,6 @@ var profile = &messageValueStruct{
 				testVal:     "1234567890",
 			},
 		},
-		&primitiveTypedField{
-			fieldName:       "StartTime",
-			originFieldName: "TimeNanos",
```

Comment:
It's called StartTime here, but TimeNanos in the proto. So yes, the field is needed.
We could possibly fix the mismatch. But we shouldn't remove it entirely.

Reply:
The field `time_nanos` is already covered by the element just before this removed one:
```
		&primitiveTypedField{
			fieldName:       "Time",
			originFieldName: "TimeNanos",
			returnType: &primitiveType{
				structName:  "Timestamp",
				packageName: "pcommon",
				rawType:     "int64",
				defaultVal:  "0",
				testVal:     "1234567890",
			},
		},
```

Not sure how `StartTime` ended up here - but it seems like it is a duplicate.

Reply:
There seems to be contrib code paths relying on it. We should change them first (especially as this is a duplicate).

Reply:
It was contrib that did put me down on this path :sweat_smile:  but I wasn't quite sure which part needed to be changed first. Will raise also the contrib change.